### PR TITLE
Update natspec that was missing for some functions

### PR DESCRIPTION
**Pull request by cducrest**
close https://github.com/trustlines-protocol/contracts/issues/52

### CODE REVIEW

**Review by dakingha69 — APPROVED** [review]
LGTM

File: contracts/currency-network/CurrencyNetworkInterface.sol [hunk 1]
```diff
@@ -23,10 +43,21 @@ interface CurrencyNetworkInterface {
     )
         external;
 
+    /**
+     * @notice returns what _b owes to _a
+     * @param _a First address that defines the trustline
+     * @param _b second address that defines the trustline
```
Comment (dakingha69) [inline contracts/currency-network/CurrencyNetworkInterface.sol:49]: Not consistent with params `_from` and `_to`. The compilation fails therefore with
```
DocstringParsingError: Documented parameter "_b" not found in the parameter list of the function.
```

File: contracts/currency-network/CurrencyNetworkInterface.sol [hunk 2]
```diff
@@ -23,10 +43,21 @@ interface CurrencyNetworkInterface {
     )
         external;
 
+    /**
+     * @notice returns what _b owes to _a
```
Comment (dakingha69) [inline contracts/currency-network/CurrencyNetworkInterface.sol:47]: Isn't it what `_from` owes to `_to`?

**Review by dakingha69 — APPROVED** [review]
Sorry overlooked failing CircleCI. Pls fix and can be merged afterwards

File: contracts/currency-network/CurrencyNetworkInterface.sol (hunk 2 above)
Comment (cducrest) [reply contracts/currency-network/CurrencyNetworkInterface.sol:47]: No, as we can see in the implementation:
https://github.com/trustlines-protocol/contracts/blob/master/contracts/currency-network/CurrencyNetworkBasic.sol#L458

**Pull request merged**